src/execution_plan/ops/op_project.c

```diff
@@ -31,7 +39,8 @@ OpBase* NewProjectOp(const AST *ast, AR_ExpNode **exps, char **aliases) {
     project->order_exp_count = 0;
     project->singleResponse = false;
     project->aliases = aliases;
-
+    project->record_len = project->exp_count + project->order_exp_count;
```

Comment:
at this point, project->order_exp_count = 0.
See ProjectInit line 67

Reply:
fixed